Resources/views/PlatformUI/shell.html.twig

```diff
@@ -59,7 +59,7 @@
     <script>
         {{ ez_platformui_yui_config( "YUI.GlobalConfig" ) }}
 
-        var CKEDITOR_BASEPATH = ALLOYEDITOR_BASEPATH = "{{ asset("bundles/ezplatformuiassets/vendors/alloy-editor/dist/alloy-editor/") }}";
+        var CKEDITOR_BASEPATH = ALLOYEDITOR_BASEPATH = "/bundles/ezplatformuiassets/vendors/alloy-editor/dist/alloy-editor/";
```

Comment:
is there some code anywhere prefixing this url with possible base path and so on or is it not needed here?

Reply:
No, there is no code which is prefixing this patch. It is used only by AlloyEditor when it loads dependencies (like `config.js`,  `styles.js` and so on).